features/facebookImport/src/components/onOffFacebookMiniStory/onOffFacebookMiniStory.css

```diff
@@ -48,19 +53,31 @@
 }
 
 .off-facebook-events-ministory-details .divide-y p {
+    font-family: "Jost Medium";
     flex: 1 1 auto;
     font-size: 12px;
 }
 
 .off-facebook-events-ministory-details .divide-y .source {
+    font-family: "Jost Medium";
     flex: 0 0 auto;
+    margin-top: 13px;
```

Comment:
Do these changes have the same origin?

Reply:
Sorry, should have made an own commit here. Just saw that the spacings where not correct and changed it alongside